system/security/plugins/jwtSecurity/jwtCache.cpp

```diff
@@ -410,14 +408,11 @@ const std::shared_ptr<JWTUserInfo> JWTUserCache::get(const std::string& userName
         {
             time_t                          savedTime;
             std::shared_ptr<JWTUserInfo>    userInfo;
-            std::shared_ptr<JWTUserInfo>    nullUserInfo;
 
             std::tie(savedTime, userInfo) = foundIter->second;
 
             if (expireTime < savedTime)
                 return userInfo;
-            else
-                return nullUserInfo;
```

Comment:
What is returned if expired? For that matter, what is returned if the map lookup fails (I missed this earlier)?

Reply:
It falls through and the function returns a nullptr.

Reply:
My bad. I forgot about the extra set of braces in the method and thought it was ending without returning, because the final return wasn't in the diff.